app/components/post_list/index.tsx

```diff
@@ -252,7 +252,6 @@ const withPosts = withObservables(['channelId', 'forceQueryAfterAppState'], ({da
                 const {earliest, latest} = postsInChannel[0];
                 return database.get<PostModel>(POST).query(
                     Q.and(
-                        Q.where('delete_at', 0),
```

Comment:
this enhanced component was removed in #5893 

Also I think we'll need to revisit how we display the `(message deleted)` posts vs removing them from the database.

Reply:
Moved the change to channel_post_list/index.ts